docs/advanced-usage.md

```diff
@@ -223,6 +223,29 @@ netty's <code>AddressResolver</code> that resolves DNS names on
 <code>Bootstrap.connect()</code> (requires netty 4.1)</p></td>
 </tr>
 <tr>
+<td><strong>Address Resolver Group</strong></td>
+<td><code>addressResolverGroup</code></td>
+<td><code>DefaultAddressResolverGroup.INSTANCE ( or netty DnsAddressResolverGroup if present)</code></td>
+</tr>
+<tr>
+<td colspan="3"><p>Since: 6.1</p>
+<p>Sets the <code>AddressResolverGroup</code> for DNS resolution. This option is only effective if
+<code>DnsResolvers#UNRESOLVED</code> is used as <code>DnsResolver</code>. Defaults to
+<code>io.netty.resolver.DefaultAddressResolverGroup#INSTANCE</code> if <code>netty-dns-resolver</code>
+is not available, otherwise defaults to <code>io.netty.resolver.dns.DnsAddressResolverGroup</code></p>
+<p>Users of DNS-based Redis-HA setups (e.g. AWS ElastiCache) might want to configure a different DNS 
+resolver group. For example: <pre>
+new DnsAddressResolverGroup(
```

Comment:
Perhaps format this as a code block?
(see line 302 for example)

Reply:
thanks. fixed accordingly. please take a look.

(sorry for late follow up)

Reply:
Awesome, thanks!